dotfiles.py

```diff
@@ -1,29 +1,52 @@
 #!/usr/bin/env python3.8
-"""Module docstring."""
+
 import filecmp
+from collections import namedtuple
+from os.path import exists
 from os.path import expanduser
 from shutil import copyfile
 from typing import List
-from typing import Tuple
+from typing import NamedTuple
 
 import click
 
-DOTFILES: List[Tuple[str, str]] = [("~/.tmux.conf", "tmux.conf"), ("~/.vimrc", "vimrc")]
+
+class Dotfile(NamedTuple):
+    system_path: str
+    repo_name: str
+
+
+DOTFILES: List[Dotfile] = [
+    Dotfile(system_path=expanduser("~/.tmux.conf"), repo_name="tmux.conf"),
```

Comment:
If you chose to use the naming and type suggestions I made above that would change this instantiation to something like
```python
Dotfile(system=Path("~/.tmux.conf").expanduser(), repo=Path("tmux.conf"))
```